recipe/meta.yaml

```diff
@@ -45,7 +45,7 @@ requirements:
     - libgomp  # [linux]
   run:
     - python
-    - scipy
+    - scipy >=1.8.0
```

Comment:
Matching the version constraint here

https://github.com/scikit-learn/scikit-learn/blob/5194440b5d41e73ff436c45e35aa1476223f753c/pyproject.toml#L11

Reply:
Do you think the `NumPy` constraint should be here too?